code/modules/surgery/organs/organ_external.dm

```diff
@@ -848,27 +853,27 @@ Note that amputating the affected organ does in fact remove the infection from t
 	return 1
 
 // I put these two next to each other to highlight that both exist. This should likely be resolved.
-/obj/item/organ/external/robotize()
+/obj/item/organ/external/robotize(company, make_tough = 0, convert_all = 1)
 	..()
 	//robot limbs take reduced damage
-	brute_mod = 0.66
-	burn_mod = 0.66
+	if(!make_tough)
+		brute_mod = 0.66
+		burn_mod = 0.66
+		fail_at_full_damage = 1
+	else
+		tough = 1
```

Comment:
I'd probably retain the `fracture` status without it doing anything; I can add an exception to this for the surgery.